saleor/graphql/core/types/common.py

```diff
@@ -281,11 +281,15 @@ class TaxType(graphene.ObjectType):
 
 
 class Job(graphene.Interface):
-    id = graphene.ID(description="ID of job.", required=True)
     status = JobStatusEnum(description="Job status.", required=True)
     created_at = graphene.DateTime(
         description="Created date time of job in ISO 8601 format.", required=True
     )
     updated_at = graphene.DateTime(
         description="Date time of job last update in ISO 8601 format.", required=True
     )
+
+    @classmethod
+    def resolve_type(cls, instance, _info):
+        MODEL_TO_TYPE_MAP = {}
```

Comment:
We should add hints what MAP should contain. The second option is add some comment to look inside `resolve_object_with_metadata_type` as reference.
```suggestion
        MODEL_TO_TYPE_MAP = {
            # <DjangoModel>: <GrapheneType> 
        }
```

Reply:
Done